packages/analytics/src/adapters/adapter.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Primitive JavaScript type used in analytics tags.
+ */
+export type Primitive = number | string | boolean | bigint | symbol | null | undefined;
```

Comment:
This type already exist in the Monitoring package. When you have two packages that use the same type, you must extract this type and declare it in the types package